sim-lib/src/lib.rs

```diff
@@ -967,18 +974,44 @@ impl PaymentResultLogger {
         }
 
         self.total_sent += details.amount_msat;
-        self.call_count += 1;
```

Comment:
I just realized that this was a `u8`, which would have made the logger task panic after 255 additions, so that's patches 😆